app/models/warehouse_report/cas_apr.rb

```diff
@@ -42,24 +42,23 @@ def unique_non_hmis_households
 
   # unique clients on ProviderOnly route
   def referred_to_rrh
-    GrdaWarehouse::CasReport.where(match_route: 'Provider Only Route').
-      started_between(start_date: self[:start_date], end_date: self[:end_date]).
+    matches_scope.
+      where(match_route: 'Provider Only Route').
       distinct.
       select(:client_id)
   end
 
   # matches that have progressed past the initial DND review phase
   def referred_to_psh
-    GrdaWarehouse::CasReport.where(match_route: 'Default Match Route').
-      started_between(start_date: self[:start_date], end_date: self[:end_date]).
+    matches_scope.
+      where(match_route: 'Default Match Route').
```

Comment:
This should probably be adjusted to take into account the Match route Four... Only problem is that we translate that , so it shows up as Main Route.... Perhaps we should add those strings to the translations in the warehouse too...  Not totally sure where this is headed yet.